mllib/src/main/scala/org/apache/spark/ml/regression/GBTRegressor.scala

```diff
@@ -173,6 +178,10 @@ object GBTRegressor extends DefaultParamsReadable[GBTRegressor] {
 
   @Since("2.0.0")
   override def load(path: String): GBTRegressor = super.load(path)
+
+  @Since("2.3.0")
```

Comment:
You removed this from the classifier but not here (as per https://github.com/apache/spark/pull/18118#discussion_r148063169)

Reply:
done